### ISSUE TITLE
[Bug]: After upgrading, IVF_SQ8 index type will have one more key `nbits` and this make creating index failed

### ISSUE THREAD

**Issue by zhuwenxing** [milvus-io/milvus#20179]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Environment

```markdown
- Milvus version: v2.1.4 --> master-20221028-19baecdc
- Deployment mode(standalone or cluster): standalone
- SDK version(e.g. pymilvus v2.0.0rc2):
- OS(Ubuntu or CentOS): 
- CPU/Memory: 
- GPU: 
- Others:
```

### Current Behavior

```
[2022-10-28T12:07:44.956Z] <name>: task_2_IVF_SQ8

[2022-10-28T12:07:44.956Z] <partitions>: [{"name": "_default", "collection_name": "task_2_IVF_SQ8", "description": ""}]

[2022-10-28T12:07:44.956Z] <description>: test collection

[2022-10-28T12:07:44.956Z] <schema>: {

[2022-10-28T12:07:44.956Z]   auto_id: False

[2022-10-28T12:07:44.956Z]   description: test collection

[2022-10-28T12:07:44.956Z]   fields: [{

[2022-10-28T12:07:44.956Z]     name: count

[2022-10-28T12:07:44.956Z]     description: 

[2022-10-28T12:07:44.956Z]     type: 5

[2022-10-28T12:07:44.956Z]     is_primary: True

[2022-10-28T12:07:44.956Z]     auto_id: False

[2022-10-28T12:07:44.956Z]   }, {

[2022-10-28T12:07:44.956Z]     name: random_value

[2022-10-28T12:07:44.956Z]     description: 

[2022-10-28T12:07:44.956Z]     type: 11

[2022-10-28T12:07:44.956Z]   }, {

[2022-10-28T12:07:44.956Z]     name: float_vector

[2022-10-28T12:07:44.956Z]     description: 

[2022-10-28T12:07:44.956Z]     type: 101

[2022-10-28T12:07:44.956Z]     params: {'dim': 128}

[2022-10-28T12:07:44.956Z]   }]

[2022-10-28T12:07:44.956Z] }

[2022-10-28T12:07:44.956Z] 

[2022-10-28T12:07:44.956Z] Traceback (most recent call last):

[2022-10-28T12:07:44.956Z]   File "scripts/action_after_upgrade.py", line 108, in <module>

[2022-10-28T12:07:44.956Z]     task_2(data_size, host)

[2022-10-28T12:07:44.956Z]   File "scripts/action_after_upgrade.py", line 40, in task_2

[2022-10-28T12:07:44.956Z]     create_index(prefix)

[2022-10-28T12:07:44.956Z]   File "/home/jenkins/agent/workspace/tests/python_client/deploy/scripts/utils.py", line 152, in create_index

[2022-10-28T12:07:44.956Z]     c.create_index(field_name="float_vector", index_params=index)

[2022-10-28T12:07:44.956Z]   File "/usr/local/lib/python3.8/dist-packages/pymilvus/orm/collection.py", line 1092, in create_index

[2022-10-28T12:07:44.956Z]     return conn.create_index(self._name, field_name, index_params,

[2022-10-28T12:07:44.956Z]   File "/usr/local/lib/python3.8/dist-packages/pymilvus/decorators.py", line 109, in handler

[2022-10-28T12:07:44.956Z]     raise e

[2022-10-28T12:07:44.956Z]   File "/usr/local/lib/python3.8/dist-packages/pymilvus/decorators.py", line 105, in handler

[2022-10-28T12:07:44.956Z]     return func(*args, **kwargs)

[2022-10-28T12:07:44.956Z]   File "/usr/local/lib/python3.8/dist-packages/pymilvus/decorators.py", line 136, in handler

[2022-10-28T12:07:44.956Z]     ret = func(self, *args, **kwargs)

[2022-10-28T12:07:44.956Z]   File "/usr/local/lib/python3.8/dist-packages/pymilvus/decorators.py", line 85, in handler

[2022-10-28T12:07:44.956Z]     raise e

[2022-10-28T12:07:44.956Z]   File "/usr/local/lib/python3.8/dist-packages/pymilvus/decorators.py", line 50, in handler

[2022-10-28T12:07:44.956Z]     return func(self, *args, **kwargs)

[2022-10-28T12:07:44.956Z]   File "/usr/local/lib/python3.8/dist-packages/pymilvus/client/grpc_handler.py", line 563, in create_index

[2022-10-28T12:07:44.956Z]     raise MilvusException(status.error_code, status.reason)

[2022-10-28T12:07:44.956Z] pymilvus.exceptions.MilvusException: <MilvusException: (code=1, message=CreateIndex failed: index already exist, but parameters are inconsistent. source index: {index_name: _default_idx_102, field_id: 102, index_params: [key:"index_type" value:"IVF_SQ8"  key:"nlist" value:"128"  key:"metric_type" value:"L2" ], type_params: [key:"dim" value:"128" ]} current index: {index_name: _default_idx_102, field_id: 102, index_params: [key:"index_type" value:"IVF_SQ8"  key:"nlist" value:"128"  key:"metric_type" value:"L2"  key:"nbits" value:"8" ], type_params: [key:"dim" value:"128" ]})>

script returned exit code 1
```

### Milvus Log

failed job: https://qa-jenkins.milvus.io/blue/organizations/jenkins/deploy_test/detail/deploy_test/546/pipeline/313
log:
[artifacts-standalone-upgrade-546-server-logs.tar.gz](https://github.com/milvus-io/milvus/files/9888697/artifacts-standalone-upgrade-546-server-logs.tar.gz)
[artifacts-standalone-upgrade-546-pytest-logs.tar.gz](https://github.com/milvus-io/milvus/files/9888699/artifacts-standalone-upgrade-546-pytest-logs.tar.gz)

**Comment by longjiquan** [comment]
already fixed by #20183 
please verify this again, thx, @zhuwenxing 
/assign @zhuwenxing

**Comment by zhuwenxing** [comment]
Verified and passed
